readme.md

```diff
@@ -50,8 +50,8 @@ Wink is shipped with an admin panel that's simple to use. However, we give you f
 public function index()
 {
     $posts = WinkPost::with('tags')
-        ->where('published', true)
-        ->where('publish_date', '<=', now()->toDateTimeString())
+        ->published()
+        ->beforePublishDate(now()->toDateTimeString())
```

Comment:
Maybe we call it `live()` and do the comparison in the scope? That way it reads `->published()->live()`

Reply:
Actually I was thinking the same, so I agree. Do you want to keep these as additional helpers or not?

Reply:
I guess we can keep it yes.